internal/cri/server/container_create_linux_test.go

```diff
@@ -597,7 +597,7 @@ func TestMountPropagation(t *testing.T) {
 			var spec runtimespec.Spec
 			spec.Linux = &runtimespec.Linux{}
 
-			err := opts.WithMounts(c.os, config, []*runtime.Mount{test.criMount}, "", nil)(context.Background(), nil, nil, &spec)
+			err := opts.WithMounts(c.os, config, []*runtime.Mount{test.criMount}, "", nil, false)(context.Background(), nil, nil, &spec)
```

Comment:
nit: `"" /*mountLabel*/, nil /*handler*/, false /*cgroupWritable*/` for slightly better readability